console.log("hello");
let a = 5;
try {
    console.log(a);
}
catch {
    console.log("caught an error... a is not defined");
}
console.log("hello2");